model/account/accounts.go

```diff
@@ -18,20 +18,25 @@ import (
 
 // Account holds configuration information for an account
 type Account struct {
-	DocID             string                 `json:"_id,omitempty"`
-	DocRev            string                 `json:"_rev,omitempty"`
-	Name              string                 `json:"name"`
-	AccountType       string                 `json:"account_type"`
-	DefaultFolderPath string                 `json:"defaultFolderPath,omitempty"`
-	FolderPath        string                 `json:"folderPath,omitempty"` // Legacy
-	Token             string                 `json:"token,omitempty"`      // Used by bi-aggregator
-	UserID            string                 `json:"user_id,omitempty"`    // Used by bi-aggregator-user
-	Basic             *BasicInfo             `json:"auth,omitempty"`
-	Oauth             *OauthInfo             `json:"oauth,omitempty"`
-	Extras            map[string]interface{} `json:"oauth_callback_results,omitempty"`
-	Relationships     map[string]interface{} `json:"relationships,omitempty"`
-	Data              map[string]interface{} `json:"data,omitempty"`
-	Metadata          *metadata.CozyMetadata `json:"cozyMetadata,omitempty"`
+	DocID         string                 `json:"_id,omitempty"`
+	DocRev        string                 `json:"_rev,omitempty"`
+	Relationships map[string]interface{} `json:"relationships,omitempty"`
+	Metadata      *metadata.CozyMetadata `json:"cozyMetadata,omitempty"`
+
+	AccountType       string                   `json:"account_type"`
+	Name              string                   `json:"name"`                        // Filled with the value of
```

Comment:
I don't understand the comment `Filled with the value of`. Is it truncated?

Reply:
That was somehow truncated, yes.